src/codeu/chat/client/simplegui/Conversation.java

```diff
@@ -0,0 +1,108 @@
+package codeu.chat.client.simplegui;
+
+/**
+ * @author  Troy Jones
+ * @date    5/21/17
+ * @brief   Contains all GUI necessary to execute the frame to 
+ *          enable chat.
+ */
+
+import java.awt.BorderLayout;
+import java.awt.Button;
+import java.awt.Container;
+import java.awt.FlowLayout;
+import java.awt.GridBagConstraints;
+import java.awt.GridBagLayout;
+import java.awt.event.MouseAdapter;
+import java.awt.event.MouseEvent;
+
+import javax.swing.BoxLayout;
+import javax.swing.JButton;
+import javax.swing.JFrame;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.JTextArea;
+import javax.swing.JTextField;
+
+@SuppressWarnings("serial")
+
+public class Conversation extends JFrame {
+
+    boolean notClickedYet = true;
+
+    // constructor
+    public Conversation() {
+
+        JFrame frame = new JFrame();
+
+        // fill mainPanel
```

Comment:
this comment gives me no new information on what "mainPanel" is or what it's supposed to do